lib/ontologies_linked_data/models/mod/semantic_artefact_distribution.rb

```diff
@@ -2,62 +2,64 @@ module LinkedData
     module Models
 
         class SemanticArtefactDistribution < LinkedData::Models::Base
-            
-            class << self
-                attr_accessor :attribute_mappings
-                def attribute_mapped(name, **options)
-                  mapped_to = options.delete(:mapped_to)
-                  attribute(name, **options)
-                  @attribute_mappings ||= {}
-                  @attribute_mappings[name] = mapped_to if mapped_to
-                end
-            end
+            include LinkedData::Concerns::SemanticArtefact::AttributeMapping
+            include LinkedData::Concerns::SemanticArtefact::AttributeFetcher
 
             model :SemanticArtefactDistribution, namespace: :mod, name_with: ->(s) { distribution_id_generator(s) }
             
             # SAD attrs that map with submission
             attribute_mapped :distributionId, mapped_to: { model: :ontology_submission, attribute: :submissionId }
             attribute_mapped :title, namespace: :dcterms, mapped_to: { model: :ontology_submission, attribute: :URI }
-            attribute_mapped :deprecated, namespace: :owl, mapped_to: { model: :ontology_submission, attribute: :deprecated }
+            attribute_mapped :deprecated, namespace: :owl, mapped_to: { model: :ontology_submission }
             attribute_mapped :hasRepresentationLanguage, namespace: :mod, mapped_to: { model: :ontology_submission, attribute: :hasOntologyLanguage }
-            attribute_mapped :hasFormalityLevel, namespace: :mod, mapped_to: { model: :ontology_submission, attribute: :hasFormalityLevel }
+            attribute_mapped :hasFormalityLevel, namespace: :mod, mapped_to: { model: :ontology_submission }
             attribute_mapped :hasSyntax, namespace: :mod, mapped_to: { model: :ontology_submission, attribute: :hasOntologySyntax }
```

Comment:
how much of the attributes in `SemanticArtefactDistribution` and in `SemanticArtefact` are shared? you can create a parent Object that can extract the shared ones, and the two model will extend it. 
will be much easier to update/edit in the future.

Reply:
there are just 6 attributes that are shared between the two 
`:title, :deprecated, :useGuidelines, :description, :metrics, :numberOfAgents`, so i think we don't need a parent object for now